run_manylinux_aarch64_build.sh

```diff
@@ -0,0 +1,90 @@
+#!/bin/bash
+__heredoc__="""
+notes:
+    Manylinux repo: https://github.com/pypa/manylinux 
+    Win + Osx repo: https://github.com/mavlink/MAVSDK-Python
+"""
+
+
+DOCKER_IMAGE=${DOCKER_IMAGE:="quay.io/pypa/manylinux2014_aarch64:latest"}
```

Comment:
Is the change as simple as changing `DOCKER_IMAGE`? Can we just pass this as an argument to the existing script?

Reply:
> Is the change as simple as changing `DOCKER_IMAGE`? Can we just pass this as an argument to the existing script?

Done.